http_handler.go

```diff
@@ -187,9 +187,9 @@ func (hh *httpHandler) PutAnnotations(w http.ResponseWriter, r *http.Request) {
 
 	tid := transactionidutils.GetTransactionIDFromRequest(r)
 	err = hh.annotationsService.Write(uuid, lifecycle, platformVersion, tid, anns)
-	if err == annotations.UnsupportedPredicateErr {
+	if errors.Cause(err) == annotations.UnsupportedPredicateErr {
```

Comment:
As of golang1.13 error wrapping is part of the standard library. So we can use `errors.Is` here instead of `github.com/pkg/errors` `errors.Cause`. 
For more info can be found here: https://blog.golang.org/go1.13-errors
```suggestion
	if errors.Is(err, annotations.UnsupportedPredicateErr) {
```